Added EventFiringWebElement to the isinstance check in the move_to me…

kmcgon opened the pull request:
Updated The isinstance check to look for WebElement or EventFiringWebElement.
This will fix cases where the move_to() method in ActionChains will throw AttributeErrors when using EventFiringWebDriver.

Fixes #6604

<!-- NOTE
Please be aware that the Selenium Grid 3.x is being deprecated in favour of the
upcoming version 4.x. We won't be receiving any PRs related to the Grid 3.x code. Thanks!
-->




- [x] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/seleniumhq/selenium/7018)
<!-- Reviewable:end -->

lmtierney commented:
Thanks for the pull request @kmcgon, can you please add the same to https://github.com/SeleniumHQ/selenium/blob/7c302be174d5c36444ee97a8712663afc8504d69/py/selenium/webdriver/common/actions/pointer_input.py#L36-L43

Then add a test for both pointer and key inputs to ensure we can use EventFiringWebDriver with each?

Such as:

```
from selenium.webdriver.support.abstract_event_listener import AbstractEventListener
from selenium.webdriver.support.event_firing_webdriver import EventFiringWebDriver
def test_can_use_pointer_input_with_event_firing_webdriver(driver, pages):
    efwd = EventFiringWebDriver(driver, AbstractEventListener())
    pages.load("javascriptPage.html")
    toClick = efwd.find_element_by_id("clickField")

    actions = ActionBuilder(efwd)
    pointer = actions.pointer_action

    pointer.move_to(toClick).click()

    actions.perform()
    assert 'Clicked' == toClick.get_attribute('value')
```

kmcgon commented:
@lmtierney Thank you for reviewing. Selenium has been good to me, I'm glad to be contributing back. I've pushed up the requested change and added a couple tests. Let me know if that's adequate coverage for this, if not I'd be happy to add more. I see I also introduced a few linter errors... I will get those cleaned up tonight.

lmtierney commented:
Looks great. Can you address flake8 and then I'll get it merged? https://travis-ci.com/SeleniumHQ/selenium/jobs/211016781

kmcgon commented:
The flake8 errors have all been address, should be all set now. Thanks @lmtierney.
